src/js/chat/templates.js

```diff
@@ -73,6 +73,12 @@ var jtvEmoticonize = exports.jtvEmoticonize = function(id) {
 };
 
 var emoticon = exports.emoticon = function(id, name) {
+    var fixedTwitchEmotes = [15, 16, 17, 18, 19, 20, 21, 22, 26, 27, 33];
+    id = parseInt(id, 10);
+
+    if (fixedTwitchEmotes.indexOf(id) > -1) {
```

Comment:
more efficient to just parseInt here.. ie `.indexOf(parseInt(id, 10))` otherwise js has to keep type converting to string